images/linux/scripts/installers/python.sh

```diff
@@ -6,18 +6,43 @@
 
 set -e
 # Source the helpers for use with the script
+source $HELPER_SCRIPTS/etc-environment.sh
 source $HELPER_SCRIPTS/os.sh
 
 # Install Python, Python 3, pip, pip3
 if isUbuntu16 || isUbuntu18; then
-    apt-get install -y --no-install-recommends python python-dev python-pip python3 python3-dev python3-pip
+    apt-get install -y --no-install-recommends python python-dev python-pip python3 python3-dev python3-pip python3-venv
 fi
 
 if isUbuntu20; then
-    apt-get install -y --no-install-recommends python3 python3-dev python3-pip
+    apt-get install -y --no-install-recommends python3 python3-dev python3-pip python3-venv
     ln -s /usr/bin/pip3 /usr/bin/pip
 fi
 
+if isUbuntu18 || isUbuntu20 ; then
+    # Install pipx
+    # Set pipx custom directory
+    export PIPX_BIN_DIR=/opt/pipx_bin
+    export PIPX_HOME=/opt/pipx
+
+    python3 -m pip install pipx
+    python3 -m pipx ensurepath
+
+    # Update /etc/environment
+    setEtcEnvironmentVariable "PIPX_BIN_DIR" $PIPX_BIN_DIR
+    setEtcEnvironmentVariable "PIPX_HOME" $PIPX_HOME
+    prependEtcEnvironmentPath /opt/pipx_bin
+
+    # Add pipx bin directory to path
+    echo 'export PATH="$PATH:/opt/pipx_bin"' >> /etc/skel/.bashrc
```

Comment:
I don't think that we need it because we have already added it in `prependEtcEnvironmentPath /opt/pipx_bin`

Reply:
Removed, thanks!